### ISSUE TITLE
5.0.9 - 'Edit Code' for a local module displays empty code

### ISSUE THREAD

**Issue by bt-ndollimount** [ironmansoftware/powershell-universal#3849]
### Version

5.0.9

### Severity

Low

### Environment

msi

### Steps to Reproduce

When attempting to edit a custom module through the PSU interface, clicking 'Edit Code' displays empty code.

### Expected behavior

```console
The module code should be displayed to be edited.
```

### Actual behavior

```console
The code is empty.
```

### Screenshots/Animations

![image](https://github.com/user-attachments/assets/fb19e32a-8bb1-469f-a3da-be744a4e643d)

**Comment by adamdriscoll** [comment]
Can you share the layout of your module? I tried this myself and it works but it is a really simple, single file module. 

![image](https://github.com/user-attachments/assets/b0506c99-e97e-404a-a33e-c89afd3d0117)

![image](https://github.com/user-attachments/assets/e874d635-05ee-4b65-996a-be4832f312e7)

The local module editor only supports this type of module.

**Comment by bt-ndollimount** [comment]
![image](https://github.com/user-attachments/assets/9ec753f7-1ddd-413c-b004-9a721b1ef8bf)

It's a module I initially created in PSU with that layout.

**Comment by bt-ndollimount** [comment]
A fun turn of events; it's now loading it. Probably one of the service restarts while trying to get the script back.

**Comment by bt-ndollimount** [comment]
@adamdriscoll This is happening again since yesterday. Is there anything you can think of for me to try for troubleshooting?

**Comment by bt-ndollimount** [comment]
When checking the browser console, this line, "Duplicate definition of module 'vs/basic-languages/powershell/powershell' pops up. I'm not sure if that's relevant.

![image](https://github.com/user-attachments/assets/b4336cbd-411f-454e-ad2a-beb5e471d666)

**Comment by adamdriscoll** [comment]
I'm not sure. Let me play around with it a bit more to see if I can reproduce it.

**Comment by adamdriscoll** [comment]
I was able to reproduce this. Should be fixed in 5.0.11
